how-to/register-with-browser/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "openfin-workspace--register-with-browser",
+  "version": "6.0",
```

Comment:
we just need to add an extra .0 so it is 6.0.0 because the github actions uses the version number for folder naming.

Reply:
added extra 0